client/submodules/live/GlassLiveConnection.cs

```diff
@@ -240,6 +240,8 @@
 	}
 
 	%data = $JSON::Value;
+	GlassGroup.add(%data);
```

Comment:
This could be fixed by always expecting the data to be removed, and always fetching raw data, no objects.